### ISSUE TITLE
Can't create the initial api object -- maybe python version isn't supported?

### ISSUE THREAD

**Issue by bgtron** [mackliet/church_of_jesus_christ_api#21]
Hi mackliet!  Your API wrapper looks really awesome and I'd love to automate some things with it...but I'm getting this error when trying to instantiate ChurchOfJesusChristAPI:

```
python3.13/site-packages/church_of_jesus_christ_api/church_of_jesus_christ_api.py", line 141, in __init__
    self.__session.get(challenge_resp["success"]["href"])
                       ~~~~~~~~~~~~~~^^^^^^^^^^^
KeyError: 'success'
```

I'm using Python 3.13...maybe that's too new?  Any ideas?  I'll try to debug, but thought I'd reach out.

**Comment by bgtron** [comment]
Oh...I have 2FA enabled...I think it is not handling the redirect properly.

**Comment by piqey** [comment]
> Oh...I have 2FA enabled...I think it is not handling the redirect properly.

Yeah. Doesn't work work with MFA. Which sucks if you're in a calling that _requires_ you to have MFA.
